### PR TITLE
Added link to github issues

### PR DESCRIPTION
**Pull request by shaedrich**
Refers to: #509

### CODE REVIEW

File: views/layout.cfm [hunk 1]
```diff
@@ -74,6 +74,7 @@
 						<li><a href="/reports/missing-descriptions.cfm">Missing Descriptions</a></li>
 						<li><a href="/reports/missing-examples.cfm">Missing Examples</a></li>
 						<li><a href="/reports/missing-related.cfm">Missing Related Links</a></li>
+						<li><a href="https://github.com/foundeo/cfdocs/issuesm">More...</a></li>
```
Comment (pfreitag) [inline views/layout.cfm]: The url should be `https://github.com/foundeo/cfdocs/issues` you have an extra `m` at the end.

Comment (shaedrich) [inline views/layout.cfm]: Silly me! Changed that.

**Comment by pfreitag** [comment]
Thanks again for all the improvements!

**Comment by shaedrich** [comment]
No problem!

That's the way open source should work. I like that it all works fuss-free with you.

**Pull request merged**